documentation/book/proc-upgrading-the-cluster-operator-0-10-0-to-0-11-0.adoc

```diff
@@ -1,87 +0,0 @@
-// This module is included in the following assemblies:
-//
-// assembly-upgrade.adoc
-
-[id='proc-upgrading-the-cluster-operator-0-10-0-to-0-11-0-{context}']
-= Upgrading the Cluster Operator from {ProductLongName} 0.10.0 to 0.11.0
```

Comment:
I guess I'm fine with removing the section about upgrading older versions. But should we add somewhere a not that for upgrading from older then previous version of the operator users should look at the older documentation? Otherwise they might miss important stuff when they try to upgrade across multiple versions.

Reply:
Absolutely. I added a note: 'NOTE: Refer to previous {ProductName} documentation for information on upgrading from earlier Cluster Operation versions.'